project/ZipkinResolver.scala

```diff
@@ -49,7 +49,13 @@ object ZipkinResolver extends Plugin {
       }) ++ Seq(
         // the local repo has to be in here twice, because sbt won't push to a "file:"
         // repo, but it won't read artifacts from a "Resolver.file" repo. (head -> desk)
-        "local-lookup" at ("file:" + localRepo.getAbsolutePath),
+        "local-lookup" at {
+          val original = "file:" + localRepo.getAbsolutePath
+          val crossPlatform = localRepo.getAbsoluteFile.toURI.toString
+          println("The original statement says that the repository url is: "+original)
```

Comment:
Are these printlns going to stay in here forever? Any reason they can't go away?